dipy/align/sumsqdiff.pyx

```diff
@@ -91,12 +113,35 @@ cdef int _solve_3d_symmetric_positive_definite(double[:] g, double[:] y, double
     return 0
 
 
-def solve_3d_symmetric_positive_definite(double[:] g, double[:] y, double tau,
-                                         double[:] out):
+def solve_3d_symmetric_positive_definite(g, y, double tau):
     r"""
     Wrapper for _solve_3d_symmetric_positive_definite (see documentation above)
```

Comment:
And by "same" I mean that a public interface wrapping a function should probably just state what the "wrapped" function does.